<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:light;
font: 75 16pt &quot;微软雅黑&quot;;
font-size:14px;
font-color:rgb(0, 0, 0);
background:white;
border: 1px solid rgb(60,163,247);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>昵称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:light;
font: 75 16pt &quot;微软雅黑&quot;;
font-size:14px;
font-color:rgb(0, 0, 0);
background:white;
border: 1px solid rgb(60,163,247);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>个性签名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:light;
font: 75 16pt &quot;微软雅黑&quot;;
font-size:14px;
font-color:rgb(0, 0, 0);
background:white;
border: 1px solid rgb(60,163,247);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:light;
font: 75 16pt &quot;微软雅黑&quot;;
font-size:14px;
font-color:rgb(0, 0, 0);
background:white;
border: 1px solid rgb(60,163,247);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:light;
font: 75 16pt &quot;微软雅黑&quot;;
font-size:14px;
font-color:rgb(0, 0, 0);
background:white;
border: 1px solid rgb(60,163,247);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>在线</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:light;
font: 75 16pt &quot;微软雅黑&quot;;
font-size:14px;
font-color:rgb(0, 0, 0);
background:white;
border: 1px solid rgb(60,163,247);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>忙碌</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:light;
font: 75 16pt &quot;微软雅黑&quot;;
font-size:14px;
font-color:rgb(0, 0, 0);
background:white;
border: 1px solid rgb(60,163,247);
border-radius:10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:light;
font: 75 16pt &quot;微软雅黑&quot;;
font-size:14px;
font-color:rgb(0, 0, 0);
background:white;
border: 1px solid rgb(60,163,247);
border-radius:10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:light;
font: 75 16pt &quot;微软雅黑&quot;;
font-size:14px;
font-color:rgb(0, 0, 0);
background:white;
border: 1px solid rgb(60,163,247);
border-radius:10px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:light;
font: 75 16pt &quot;微软雅黑&quot;;
font-size:14px;
font-color:rgb(0, 0, 0);
background:white;
border: 1px solid rgb(60,163,247);
border-radius:10px;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{ 
     background-color: rgb(6,157,213); 
     border: none;
     border-radius:10px;
     color:white;
}
QPushButton:hover{ 
     background-color: rgb(75,202,255);
}</string>
   </property>
   <property name="text">
    <string>开始进入联系人管理界面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{ 
     background-color: rgb(6,157,213); 
     border: none;
     border-radius:10px;
     color:white;
}
QPushButton:hover{ 
     background-color: rgb(75,202,255);
}</string>
   </property>
   <property name="text">
    <string>已注册，开始登陆</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
